pkg/devicetwin/dtcontroller.go

```diff
@@ -77,8 +77,9 @@ func (dtc *DTController) Start() error {
 			if msg, ok := dtc.DTContexts.ModulesContext.Receive("twin"); ok == nil {
 				log.LOGGER.Info("DeviceTwin receive msg")
 				err := dtc.distributeMsg(msg)
-				// todo  deal error
 				if err != nil {
+					log.LOGGER.Infof("distributeMsg failed: %v", err)
```

Comment:
please add a test for this case and validate it works fine.

Reply:
Have added the UT in the same PR.